duckduckgo/iOS
# Feature/update private practices statement

Reviewer: Caine or Mia
Asana: https://app.asana.com/0/414235014887631/495876713362560
CC:

**Description**:

Update privacy statement logic and add tests for it

**Steps to test this PR**:

Visit the following websites and ensure text is correct on the overview and privacy practices page

* falkirkrpg.org.uk - "Unknown Privacy Practices
* google.com - "Poor Privacy Practices"
* soundcloud.com - "Mixed Privacy Practices"
* duckduckgo.com - "Good Privacy Practices"
